app/services/PaypalService.scala

```diff
@@ -87,12 +90,14 @@ class PaypalService(config: PaypalApiConfig, contributionData: ContributionData)
       createdPayment.getLinks.asScala
     } match {
       case Success(links) =>
-        val approvalLink = links.find(_.getRel.equalsIgnoreCase("approval_url"))
-        approvalLink.map(l => Right(l.getHref)).getOrElse(Left("No approval link returned from paypal"))
+        val approvalLink = links.find(_.getRel.equalsIgnoreCase("approval_url")).map(l => Right(addUserActionParam(l.getHref)))
+        approvalLink.getOrElse(Left("No approval link returned from paypal"))
       case Failure(exception) => Left(exception.getMessage)
     }
   }
 
+  private def addUserActionParam(url:String) = Uri.parse(url).addParam("useraction", "commit").toString
```

Comment:
Aparently there is no API parameter to change the button from "continue" to "pay now" we have to manually add a query parameter to the returned url.